server/package.json

```diff
@@ -1,16 +1,22 @@
 {
   "name": "server",
+  "description": "Node.js REST API with express, mysql and sequelize",
```

Comment:
This should be the project overview what it does generally

Reply:
Oh,, didn't know

Reply:
just a brief description

Reply:
ok..would make necessary changes